packages/component/src/Typography/Typography.tsx

```diff
@@ -0,0 +1,19 @@
+import { TypographyProps } from '@blue-learn/schema';
+import React, { memo } from 'react';
+import { TypographyBase } from './Typography.base';
+
+/**
+ * Primary UI component for user interaction
+ */
+
+const Typography: React.FunctionComponent<
+	TypographyProps
+> = ({ ...props }) => {
+	/**
+	 *  logic -> base-component(...props)
+	 * */
+	return <TypographyBase value={''} {...props} />;
```

Comment:
value should be also part of {...props} don't modify value here. 
Props since we are already handled with default value in base